core/templates/map/map.html

```diff
@@ -148,7 +187,8 @@
                     'text-justify': 'left',
                     'text-allow-overlap': true,
                     'text-size': 14,
-                    'text-offset': [0, 0.9],
+                    'text-offset': [0, 0],
+                    'text-anchor': 'top',
```

Comment:
Seems Maplibre's default is a center text-anchor. This caused some overlap with the icon for longer labels:
<img width="153" alt="Screenshot 2025-03-30 at 9 48 46 AM" src="https://github.com/user-attachments/assets/0486d287-d4de-48fa-90e6-21b8a25c1a95" />

After changing text anchor to top:
<img width="153" alt="Screenshot 2025-03-30 at 9 49 32 AM" src="https://github.com/user-attachments/assets/becdb4bb-4ef7-4cdf-b1ca-580d7e113ae4" />

Reply:
stellar, for real!